portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,6 +86,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String addedBy = (String) entity.getProperty("addedBy");
       Date addedDate = (Date) entity.getProperty("addedDate");
 
+      // If a languageCode was specified, translate the comments' message in that language
+      if(languageCode != null) {
+        Translate translate = TranslateOptions.getDefaultInstance().getService();
+        Translation translation =
+            translate.translate(message, Translate.TranslateOption.targetLanguage(languageCode));
+        message = translation.getTranslatedText();
```

Comment:
Is there an assumption that the original message is in English? What if it is not?

Reply:
No, there is no assumption of the message's language. The API automatically detects it and translates it in the target language.

Reply:
Got it, thanks!